addons/libs/images.lua

```diff
@@ -244,6 +246,15 @@ function images.pos_y(t, y)
     t:pos(meta[t].settings.pos.x, y)
 end
 
+function images.last_pos(t, x, y)
```

Comment:
Created this function in order to A) expose this local variable to addons and to B) imitate the get/set of the existing pos methods: _images.pos(), images.pos_x(), and images.pos_y()_.

Reply:
Addons should not be able to get that information at all outside of the event itself. Even then it's debatable.

Reply:
Drag events are probably going to be used to compute, and then act on, said delta, so it may be a great place to make it available.